.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,90 @@
+name: Publish Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-helm-list:
+    runs-on: ubuntu-latest
+    outputs:
+      helmcharts: ${{ steps.helm-list.outputs.json }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - id: helm-list
+        run: |
+          ORG_DIR=helm-charts
```

Comment:
This isn't quite right as there's no `helm-charts` dir in the root of the repo https://github.com/osg-htc/helm-charts.

In the images repo, we were overloading the image list as A. the path to find the contents of each image, e.g. https://github.com/opensciencegrid/images/tree/main/opensciencegrid/sssd as well as B. the destination where to push the image, e.g. hub.opensciencegrid.org/**opensciencegrid/sssd**. In this repo, we have the complication of the leading "supported" vs "contrib" directories.

We still need all of (A) in the subsequent job to find the source of the build and we can still extract (B) from (A) to determine where the build needs to be pushed so I think the thing to do is populate the output of this job with the paths and then extract what you need in the subsequent job. In other words:

1. https://github.com/osg-htc/helm-charts/pull/4/files#diff-dce247c65e791dfbb2975e023ffc0d20d556f48be798bc6c462ba808ccbbc26fR38-R50 should result in full paths, e.g. `supported/osg-htc/osdf-origin/`
2. [This job](https://github.com/osg-htc/helm-charts/pull/4/files#diff-dce247c65e791dfbb2975e023ffc0d20d556f48be798bc6c462ba808ccbbc26fR85) should know how to get and use (A) and (B) from the paths that it's fed

Reply:
Ah that's the complication with this repo when compared to the `images` repo. The full path of the location of the helm chart (e.g., `supported/opensciencegrid/osdf-origin/`) is not the same as the location of where we're going to be pushing the chart (e.g., `hub.opensciencegrid.org/opensciencegrid/osdf-origin`)